presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -62,6 +62,16 @@ Array Functions
     Sorts and returns the array ``x``. The elements of ``x`` must be orderable.
     Null elements will be placed at the end of the returned array.
 
+.. function:: array_sort(array<T>, function<T,T,bigint>) -> array<T>
+
+    Sorts and returns the ``array`` based on the given comparator ``function``. The comparator will take
+    two arguments representing two elements of the ``array``. It returns a negative ``bigint``, zero,
+    or a positive ``bigint`` as the first argument is less than, equal to, or greater than the second argument.
+    Null elements will be placed at the end of the returned array. ::
```

Comment:
You should document the following:
- is the comparison function invoked on null elements?
   e.g. will `array_sort(ARRAY[1,null], (x,y) -> x / COALESCE(y, 0))` fail with division by zero?
- what happens when elements are null but the comparison function returns null for some comparisons?
  I am not sure what should be the expected behavior in this case, since comparison function needs to be transitive.